charts/prometheus-memcached-exporter/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: prometheus-memcached-exporter
 description: Prometheus exporter for Memcached metrics
 type: application
-version: 0.1.0
+version: 0.1.1
```

Adding new features, e.g. new fields, needs bumping the minor version number.

```suggestion
version: 0.2.0
```

---

thanks a lot for reviewing! I'll change that.